packages/mwp-app-route-plugin/src/route.js

```diff
@@ -10,30 +10,29 @@ export const onPreResponse = {
 	 * - In dev, it transforms the generic 500 error response JSON into a full dev-
 	 *   friendly rendering of the stack trace.
 	 */
-	method: (request: HapiRequest, reply: HapiReply) => {
+	method: (request: HapiRequest, h: HapiResponseToolkit) => {
 		const response = request.response;
 
 		if (!response.isBoom || process.env.NODE_ENV === 'production') {
-			return reply.continue();
+			return h.continue;
 		}
 		const error = response;
 		const { RedBoxError } = require('redbox-react');
 		const errorMarkup = ReactDOMServer.renderToString(
 			React.createElement(RedBoxError, { error })
 		);
-		const errorResponse = reply(
-			`<!DOCTYPE html><html><body>${errorMarkup}</body></html>`
-		);
-		errorResponse.code(error.output.statusCode);
-		return errorResponse;
+
+		return h
+			.response(`<!DOCTYPE html><html><body>${errorMarkup}</body></html>`)
+			.code(error.output.statusCode);
```

Comment:
this might need to be changed to `response.statusCode`

Reply:
i don't think so, because the response is a `Boom` object at this point

Reply:
ah okay, I didn't realize that. cool 👍